src/routes/+layout.ts

```diff
@@ -1 +1,9 @@
 export const prerender = true;
+
+// https://kit.svelte.dev/docs/load#layout-data
+// https://kit.svelte.dev/docs/load#making-fetch-requests
+export async function load({ fetch }) {
```

Comment:
This function doesn't look in the right place. Shouldn't it be next to the other API queries?

Reply:
I've linked docs, it must be there

Reply:
I can not find why the load function needs to be in this file and use fetch instead of the query library like the other API calls.

Reply:
there was a cli warning telling me to do the `fetch` request either in the `onMount` fn or in a `load` fn. I've documented myself, and since the banner should be shown across the whole website, [this is the easiest, shortest and most readable way of doing so](https://kit.svelte.dev/docs/load#layout-data)

Reply:
resolved in 0560498